WebHostLib/templates/siteMap.html

```diff
@@ -25,6 +25,7 @@ <h2>Base Pages</h2>
             <li><a href="/tutorial">Tutorials Page</a></li>
             <li><a href="/user-content">User Content</a></li>
             <li><a href="/weighted-settings">Weighted Settings Page</a></li>
+            <li><a href="{{ url_for('check')}}">Settings Checker</a></li>
```

Comment:
This already exists on the sitemap as Settings Validator under /mysterycheck

Reply:
ah, carp